### PR TITLE
[Intents] Update up to Xcode 9.2 Beta 2

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/intents.cs [hunk 1]
```diff
@@ -7818,10 +7818,16 @@ interface INSearchForAccountsIntentResponse {
 	[BaseType (typeof (INIntent))]
 	interface INSearchForNotebookItemsIntent {
 
+		[Deprecated (PlatformName.WatchOS, 4, 2, message: "Use the 'notebookItemIdentifier' .ctor instead.")]
+		[Deprecated (PlatformName.iOS, 11, 2, message: "Use the 'notebookItemIdentifier' .ctor instead.")]
```
Comment (VincentDondain) [inline src/intents.cs]: Let's not use `.ctor` we should assume not everyone knows what it means.

Also maybe we could be more specific on the parameter so something like: `Use the constructor with `string notebookItemIdentifier` instead.`

**Comment by dalexsoto** [comment]
@VincentDondain thanks it is now fixed!

**Comment by dalexsoto** [comment]
Failure: https://github.com/xamarin/maccore/issues/574

**Comment by dalexsoto** [comment]
build

**Comment by dalexsoto** [comment]
build hang (known issue) https://github.com/xamarin/maccore/issues/5

**Pull request merged**